runtime/gc_vlhgc/IncrementalGenerationalGC.cpp

```diff
@@ -2234,6 +2247,23 @@ MM_IncrementalGenerationalGC::reportGCIncrementEnd(MM_EnvironmentBase *env)
 	}
 
 	stats->_endTime = j9time_hires_clock();
+	stats->_stallTime = static_cast<MM_CycleStateVLHGC*>(env->_cycleState)->_vlhgcIncrementStats.getTotalStallTime();
+	
+	MM_VLHGCIncrementStats temp = static_cast<MM_CycleStateVLHGC*>(env->_cycleState)->_vlhgcIncrementStats;
+	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
+	omrtty_printf("!@ Trigger\n");
+	omrtty_printf("!@ _copyForwardStats\t%d\n",temp._copyForwardStats.getStallTime());
+	omrtty_printf("!@ _markStats\t%d\n",temp._markStats.getStallTime());
+	omrtty_printf("!@ _workPacketStats\t%d\n",temp._workPacketStats.getStallTime());
+	omrtty_printf("!@ _moveStallTime\t%d\n",temp._compactStats._moveStallTime);
+	omrtty_printf("!@ _rebuildStallTime\t%d\n",temp._compactStats._rebuildStallTime);
+	omrtty_printf("!@ sum\t%d\n",
+	temp._copyForwardStats.getStallTime()+
+	temp._markStats.getStallTime()+
+	temp._workPacketStats.getStallTime()+
+	temp._compactStats._moveStallTime+
+	temp._compactStats._rebuildStallTime
+	);
```

Comment:
a simple validation check: in GMP increment you should never see non-zero stats from _copyForwardStats

Reply:
@gza060625 added some validation asserts as part of testing, this being one of them. They were removed after testing, Should they added?

Reply:
yes, it's ok to add some asserts